modules/knative-serving-advanced-config.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+// * /serverless/installing_serverless/serverless-install-config-options.adoc
+
+[id="knative-serving-advanced-config_{context}"]
+= Knative Serving supported installation configuration options
+
+This guide provides information for cluster administrators about advanced installation configuration options for Knative Serving.
+
+The YAML configuration is shown by default when you are creating Knative Serving custom resource definitions. However, you can access the form view by selecting *Edit Form* in the top right corner of the *Create Knative Serving* page.
```

Comment:
I think starting from OCP 4.5 the form view is the default now.

Reply:
Added separate PR for this for 4.5+
https://github.com/openshift/openshift-docs/pull/24007